images/rabbitmq/configs/latest.apko.yaml

```diff
@@ -7,8 +7,18 @@ contents:
     - rabbitmq-server
     - wolfi-baselayout
     - bash
+    - glibc-locale-en
+    - tzdata
+    # - procps do we need this?
+    # - su-exec do we need this?
 
 paths:
+  - path: /var/lib/rabbitmq
+    type: directory
+    permissions: 0o777
+    uid: 999
```

Comment:
Interestingly I noticed we're using gid and uid 999 in this image, other images tend to use `65532` and the rabbit readme mentions `65532` too.  

https://github.com/chainguard-images/images/tree/main/images/rabbitmq#users-and-directories

Should we switch this?

Reply:
Yes please if it doesn't break anything.

Reply:
ok switched and tested locally - seems fine
```
  ##  ##      RabbitMQ 3.11.5
  ##  ##
  ##########  Copyright (c) 2007-2022 VMware, Inc. or its affiliates.
  ######  ##
  ##########  Licensed under the MPL 2.0. Website: https://rabbitmq.com

  Erlang:      25.2 [jit]
  TLS Library: OpenSSL - OpenSSL 3.0.7 1 Nov 2022
  Release series support status: supported

  Doc guides:  https://rabbitmq.com/documentation.html
  Support:     https://rabbitmq.com/contact.html
  Tutorials:   https://rabbitmq.com/getstarted.html
  Monitoring:  https://rabbitmq.com/monitoring.html

  Logs: /var/log/rabbitmq/rabbit@b934a941fe1f.log
        /var/log/rabbitmq/rabbit@b934a941fe1f_upgrade.log
        <stdout>

  Config file(s): (none)

  Starting broker... completed with 0 plugins.

```